### DISCUSSION TITLE
Release 0.37?

### DISCUSSION THREAD

**Discussion by Lawouach** [open-telemetry/opentelemetry-python-contrib#1646] [Q&A] [answered]
Hello all,

Using these packages is always such a great experience. Yet today I ran into an issue with this [issue](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1435) which is solving a problem when using the ASGI instrumentation along side the Google Cloud propagator.

I've noticed the cadence is a monthly release (roughly) so may I expect a new one soon by any chance? 

Thanks,

**Comment by srikanthccv** [comment, marked as answer]
You can expect the release soon. This week or early next week, based on the team availability and a couple of important fixed that we planned to include in the release.

**Comment by Lawouach** [comment]
Fantastic. This is such good news! Thank you :)

**Comment by lzchen** [comment]
https://github.com/open-telemetry/opentelemetry-python-contrib/releases/tag/v1.16.0

**Comment by Lawouach** [comment]
Terrific! Thank you
